test/input/check-nexus.json

```diff
@@ -1,13 +1,13 @@
 {
         "source": {
-            "name": "test_package1",
```

Comment:
I am also saying this beacuse this name change doesn't seems to be required... this is a valid configuration controlled by: `'name_must_match': True/False`

Reply:
Don't bother with their fly tests... That is not our Concourse deployment and doesn't concern us... As I wrote look at the ci/ dir for our pipeline and tests... This is what you care about...

Please revert the change to these 2 files..

Reply:
Done. But still, I did this change because I was confused with the name of the package uploaded to nexus. 
If it's done automatically (`-` to `_`)  why do we keep a "wrong" package name? 

As I wrote, the readme is really not clear about tests... 
And I haven't yet run the test with fly... (even with our ci/ dir)

Reply:
You are confusing the Python package name and what is configured in Concourse pipeline... This is a valid configuration... if the name between Python package and Concourse resource source configuration don't match we have: 'name_must_match': True/False flag to control it... If this bothers you search our org Github for: `name_must_match` and see when and how is used.

Reply:
See:
https://github.com/cf-platform-eng/concourse-pypi-resource/issues/32
https://github.com/cf-platform-eng/concourse-pypi-resource/pull/41

if you want to know more... FYI manuel-cc was a member of the PCI team at the time :)

Reply:
But i'm not talking about Concourse pipeline... I only ran nexus-integrationl.py  and all references there are with underscore `test_package1`.

My question is just: `why the uploaded package is name `test-package1` with `-` ?

Reply:
We are just loosing time like this... Lets have a video chat...